cli/browser-compatibility-table.js

```diff
@@ -0,0 +1,22 @@
+export function fixMdnURL(document) {
+  // Loop over, and mutate, all 'browser_compatibility' sections
+  document.body
+    .filter(section => section.type === "browser_compatibility")
+    .forEach(section => {
+      Object.entries(section.value.data).forEach(([key, block]) => {
+        if (!!block.__compat) {
+          block = block.__compat;
+        }
+        if (!!block.mdn_url) {
+          block.mdn_url = mutateURL(block.mdn_url);
+        }
+      });
+    });
+}
+
+function mutateURL(mdn_url) {
```

Comment:
Can you think of a better name? I know this is a "private" function but to begin with it doesn't mutate the string. It returns a *new* string.

Reply:
Also, let's use camelCase as much as possible. You call the variable/argument `url` but perhaps `absUrl`?